dapps/vue-dapp-auth/nuxt.config.ts

```diff
@@ -0,0 +1,32 @@
+import { version as authClientVersion } from '@walletconnect/auth-client/package.json'
+
+// https://v3.nuxtjs.org/api/configuration/nuxt.config
+export default defineNuxtConfig({
+  modules: [
+    '@nuxtjs/tailwindcss',
+    '@nuxtjs/color-mode',
+    'nuxt-icon',
+    '@vueuse/nuxt',
+    '@pinia/nuxt',
+  ],
+
+  tailwindcss: {
+    cssPath: '~/assets/scss/tailwind.scss',
+  },
+
+  runtimeConfig: {
+    public: {
+      /**
+       * Override in .env
+       * NOTE:
+       * - Could use Nuxt's native auto-injected NUXT_* env vars, but framework-specific prefixes are not cool
```

Comment:
Appreciate the context here, but could we simplify down to just what is being done (i.e. "avoiding framework-specific env vars") without personal opinions please